### DISCUSSION TITLE
Welcome to guibot Discussions!

### DISCUSSION THREAD

**Discussion by pevogam** [intra2net/guibot#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ndinev** [comment]
Have you seen SikuliX? Is it possible to have IDE for guibot similar to SikuliX IDE? The major problem it solves is creating and managing UI maps of elements. I have been using it for 13 years and I'm really happy with its simple functionality. I can elaborate more if this topic is interesting for anyone

**Reply by pevogam** [reply]
> Have you seen SikuliX?

Yes, in fact some of the original code base was written by a team mate that was highly inspired by SikuliX.

> Is it possible to have IDE for guibot similar to SikuliX IDE?

So far everything has been highly based on scripting but we also have an open issue about it: https://github.com/intra2net/guibot/issues/67

> The major problem it solves is creating and managing UI maps of elements. I have been using it for 13 years and I'm really happy with its simple functionality. I can elaborate more if this topic is interesting for anyone

For sure it is interesting, would you mind even sharing any of your observations about this UI element maps as a comment to the issue there? We could really benefit from collecting more good design ideas early on there.
